third_party/lua-yaml/lyaml.h

```diff
@@ -12,6 +12,9 @@ struct luaL_serializer;
 LUALIB_API int
 luaopen_yaml(lua_State *L);
 
+/** String with single or double newline char for compat option toggler. */
+extern char *FORCE_LITERAL_SUBSTRING;
```

Comment:
Why not just use `extern const char *FORCE_LITERAL_SUBSTRING` here and in `compat.c`? Then you can drop casts to `char *`.

Reply:
Done.